bl_source/bl_launch.c

```diff
@@ -20,11 +20,36 @@ bool start_application(void) {
     return false;
 }
 
+bool copy_golden_image() {
```

Comment:
I don't suppose we would ever run the golden image in place because, for example, the place we were trying to copy to was bad? i.e. if that bad place is RAM then the image can't do anything anyway.

Reply:
I decided to make it this way so that it was easier to keep track of the linkage for different binaries. The binary would have to be linked to the golden image location, meaning we would have releases with 6 binaries. If a region of the application image goes bad we do have wiggle room in the application image location so we could link it against somewhere else in its flash range and install it there no problem